### DISCUSSION TITLE
microG_GSync_v2.5.01

### DISCUSSION THREAD

**Discussion by ozingi** [taoaoooo/microG_playStore_GCSync#10] [Announcements]
## 更新日志 2022.11.30
1. microG Services Core更新到0.2.25.223616版本；

## 近期主要更新 2022.10.18
1. 针对安卓13优化,对于安卓13之前的设备请继续使用历史版本；
2. play store更新到32.3.16版本。

<hr /><em>This discussion was created from the release <a href='https://github.com/ozingi/microG_GSync/releases/tag/release_v2.5.01'>microG_GSync_v2.5.01</a>.</em>

**Comment by SnowindMe** [comment]
不更新一下？

**Reply by ozingi** [reply]
> 不更新一下？

会的，I'm sorry
